scripts/merge_geojson/merge_geojson.py

```diff
@@ -0,0 +1,210 @@
+        feature["id"] = "relation/" + osm_id
+    # version=2 is required, type=2 is for osc base64 encoded (i.e. geometry
+    # changes). type=1 is for tag fixes
+    cooperativeWork = {"meta": {"version": 2, "type": 1}}
+    operations = []
+    for osm_object in properties["fix_suggestions"]:
+        ignore_keys = [
+            "last_edit_version",
+            "last_edit_user_name",
+            "last_edit_user_id",
+            "last_edit_time",
+            "last_edit_changeset",
+        ]
+        fix_suggestions = properties["fix_suggestions"][osm_object]
+        # Cooperative challenges supports setTags and unsetTags
+        if fix_suggestions["type"] in ["UPDATE"]:
+            add_tags = {}
+            del_tags = []
+            for descriptor in fix_suggestions["descriptors"]:
+                if (
+                    descriptor["name"] == "TAG"
+                    and descriptor["key"] not in ignore_keys
+                ):
+                    if descriptor["type"] in ["ADD", "UPDATE"]:
+                        add_tags[descriptor["key"]] = descriptor["value"]
+                    elif descriptor["type"] in ["REMOVE"]:
+                        del_tags.append(descriptor["key"])
+                elif descriptor["name"] not in ["TAG"]:
+                    logging.error(
+                        f'Fixes for {descriptor["name"]} are not yet supported'
+                    )
+                    logging.debug(str(descriptor))
+            if len(add_tags) > 0:
+                operations.append({"operation": "setTags", "data": add_tags})
+            if len(del_tags) > 0:
+                operations.append({"operation": "unsetTags", "data": del_tags})
+        else:
+            # We currently don't support "ADD" or "REMOVE" for objects
+            logging.error(f'Unknown type {fix_suggestions["type"]}')
+            logging.debug(str(fix_suggestions))
+    cooperativeWork["operations"] = [
+        {
+            "operationType": "modifyElement",
+            "data": {"id": feature["id"], "operations": operations},
+        }
+    ]
+    feature_collection = geojson.FeatureCollection([feature])
+    if len(operations) > 0:
+        # cooperativeWork goes in the main FeatureCollection body
+        feature_collection["cooperativeWork"] = cooperativeWork
+        # properties["cooperativeWork"] = cooperativeWork
+        del properties["fix_suggestions"]
+    return feature_collection
+
+
+def write_feature_rfc7464(file_name: str, feature: geojson.Feature):
+    """
+    Create an RFC7464 compliant file for use with MapRoulette
+    """
+    fc = (
+        fix_suggestions_to_cooperativeWork(feature)
+        if "fix_suggestions" in feature["properties"]
+        else geojson.FeatureCollection([feature])
+    )
+    with open(file_name, "ab") as fh:
+        # Line-by-line uses the 1E byte record separator
+        fh.write(ASCII_RECORD_SEPARATOR)
+
+    with open(file_name, "a") as fh:
+        string = geojson.dumps(fc)
```

Comment:
I am guessing this is just calling json.dumps, in which case we probably want to set `ensure_ascii=False` to maintain unicode characters.